src/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 
 IF(BUILD_BULLET3)
-	SUBDIRS(  Bullet3OpenCL Bullet3Serialize/Bullet2FileLoader Bullet3Dynamics Bullet3Collision Bullet3Geometry )
+	SUBDIRS(  Bullet3Serialize/Bullet2FileLoader Bullet3Dynamics Bullet3Collision Bullet3Geometry )
+	IF(BUILD_BULLET3_OPENCL)
+		add_subdirectory ( Bullet3OpenCL )
```

Comment:
The order here might matter, so I would put it above the SUBDIRS call. SUBDIRS is also deprecated since CMake 3.0, so I guess you could also convert the whole list to add_subdirectory, but I don't know the maintainers of this project, so I'm not sure if they would want this.